src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+/**
+ * <h>duke.Parser</h>
+ * Deals with making sense of the user command.
+ */
+public class Parser {
```

Comment:
One big suggestion from me would be to expand on what your parse class does. So far from what I can see its more of a String trimmer or splitter which is quite a basic functionality. Think about how you could build on it like using the parser to get your LocalDateTime object from input and so on.